src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -1270,6 +1286,22 @@ InferenceProfiler::ValidLatencyMeasurement(
         }
       }
     }
+    const auto& response_times = std::get<1>(timestamp);
+    // Gathering durations between responses. Need more than 1 element for
+    // metrics to be valid. The invariant for timestamps is that if the final
+    // response is a null response, it will NOT be added to the timestamp vector
+    if (response_times.size() > 1) {
+      for (size_t i = 0; i < response_times.size() - 1; i++) {
+        uint64_t prev_response_ns = CHRONO_TO_NANOS(response_times[i]);
```

Comment:
nit, can you call one of these current and the other one either previous or next?

Reply:
done